packages/react-ui-core/src/Button/ApplyButton.js

```diff
@@ -52,7 +52,7 @@ export default class ApplyButton extends PureComponent {
         data-tid="apply-button"
         {...props}
       >
-        {name}
```

Comment:
Are there any concerns about breaking anything in rent or ag if they are passing `name` instead of `text`?

Reply:
Yeah I am dumb I will revert to name, I just didnt like the prop name tbh

Reply:
I'd prefer it be children instead, but we do need to check where it's being used.  If we tag the BREAKING CHANGES section through `git cz`, then we should be fine 👍

Reply:
Children it is 🎉

Reply:
children makes way more sense for this